src/ngx-gallery-preview.component.ts

```diff
@@ -116,12 +129,12 @@ export class NgxGalleryPreviewComponent implements OnChanges {
     }
 
     open(index: number): void {
-
         this.onOpen.emit();
 
         this.index = index;
         this.isOpen = true;
         this.show(true);
+        this.showNext();
```

Comment:
Why is this necessary? Seems a little bit odd, that we call `showNext`, when we just called `open`